test/parallel/test-fs-open-flags.js

```diff
@@ -60,25 +60,21 @@ assert.strictEqual(stringToFlags('xa+'), O_APPEND | O_CREAT | O_RDWR | O_EXCL);
   .forEach(function(flags) {
     assert.throws(
       () => stringToFlags(flags),
-      new RegExp(`^Error: Unknown file open flag: ${escapeRegExp(flags)}`)
+      common.expectsError({ code: 'ERR_INVALID_OPT_VALUE', type: TypeError })
     );
   });
 
 assert.throws(
   () => stringToFlags({}),
-  /^Error: Unknown file open flag: \[object Object\]$/
+  common.expectsError({ code: 'ERR_INVALID_OPT_VALUE', type: TypeError })
```

Comment:
A more specific error here would likely be better.
`ERR_FS_UNKNOWN_FILE_OPEN_FLAG` or something similar